components/style/properties/gecko.mako.rs

```diff
@@ -2397,6 +2397,7 @@ fn static_assert() {
         self.copy__x_lang_from(other)
     }
 
+    ${impl_simple("_moz_script_level", "mScriptLevel")}
```

Comment:
Is this necessary? I think it may not be, given the `gecko_ffi_name` in the `predefined_type` call.

Reply:
Yes, I think this is necessary for `set__moz_script_level`, `reset__moz_script_level` and `copy__moz_script_level` methods.
I tried to remove this line but it will throw the errors about no method names of above methods.

Reply:
Huh, ok then.